src/button/buttonview.js

```diff
@@ -90,6 +94,13 @@ export default class ButtonView extends View {
  */
 
 /**
+ * The type of the button. Default `button`.
```

Comment:
It'd be nice to say here that other possible types correspond with HTML, which is "submit", etc.